Artem.Arkhytiuk/Lesson2/index.css

```diff
@@ -0,0 +1,502 @@
+ * 2. Correct the outline style in Safari.
+ */
+
+[type="search"] {
+  -webkit-appearance: textfield; /* 1 */
+  outline-offset: -2px; /* 2 */
+}
+
+/**
+ * Remove the inner padding in Chrome and Safari on macOS.
+ */
+
+[type="search"]::-webkit-search-decoration {
+  -webkit-appearance: none;
+}
+
+/**
+ * 1. Correct the inability to style clickable types in iOS and Safari.
+ * 2. Change font properties to `inherit` in Safari.
+ */
+
+::-webkit-file-upload-button {
+  -webkit-appearance: button; /* 1 */
+  font: inherit; /* 2 */
+}
+
+/* Interactive
+   ========================================================================== */
+
+/*
+ * Add the correct display in Edge, IE 10+, and Firefox.
+ */
+
+details {
+  display: block;
+}
+
+/*
+ * Add the correct display in all browsers.
+ */
+
+summary {
+  display: list-item;
+}
+
+/* Misc
+   ========================================================================== */
+
+/**
+ * Add the correct display in IE 10+.
+ */
+
+template {
+  display: none;
+}
+
+/**
+ * Add the correct display in IE 10.
+ */
+
+[hidden] {
+  display: none;
+}
+
+nav,
+header,
+footer,
+main,
+figure,
+article,
+footer{
+  display: block;
+}
+
+body{
+  margin: 0;
+  }
+
+  .row{
+    width:100%;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+}
+
+.col-1{width: 8.33%}
+.col-2{width: 16.66%}
+.col-3{width: 25%}
+.col-4{width: 33.33%}
+.col-5{width: 41.66%}
+.col-6{width: 50%}
+.col-7{width: 58.33%}
+.col-8{width: 66.66%}
+.col-9{width: 75%}
+.col-10{width: 83.33%}
+.col-11{width: 91.66%}
+.col-12{width: 100%}
+
+
+/*##########################Navbar#################################*/
+
+header.pagehead{
+  width: 100%;
+  overflow: hidden;
+  position: fixed;
+  z-index: 1000;
+}
+
+header.pagehead nav{
+  background-color: #c7c9cc; 
+}
+
+header.pagehead nav a{
+  text-decoration: none;
+  padding: 10px;
+  display: inline-block;
+  font-size: 17px;
+  text-transform: uppercase;
+  color: #818384;
+}
+
+header.pagehead nav a:hover{
+  background-color: white;
+  color: black;
+}
+
+figure.topfigure{
+  margin: 0;
+  overflow: hidden;
+  
+}
+
+/*##########################Article#################################*/
+/* main{
+  margin-left: 10px;
+} */
+
+article{
+  line-height: 1.5;
+  padding-left: 3%;
+  padding-right: 3%;
+}
+
+article header h1{
+  text-align: center;
+  font-family: 'Mina', sans-serif;
+}
+
+article h2{
+  font-family: 'Mina', sans-serif;
+  text-align: center;
+}
+
+article p{
+  font-family: 'Crimson Text', serif;
+  font-size: 20px;
+  text-align: justify;
+}
+
+main article img{
+  margin: 10px;
+  height: 350px;
+}
+
+main article img.leftImg{
+  float: left;
+}
+
+main article img.rightImg{
+  float: right;
+  position: relative;
+  height: 150px;
+}
+
+article div.monosBlock {
+  background-color: #f1f1f1;
+  box-shadow:  0 0 0 1px rgba(0,0,0,.15);
+  border-radius: 5px;
+  width: 615px;
+  margin-left: 10px;
+  
+}
+
+article div.monosBlock pre{
```

Comment:
Too specific. Just use classnames.
Also fullnames.
monosBlock can mean a lots of shit.
monospacedText looks like a better more readable name.

Same for leftfoot. Should be leftFooter etc. More explicit names are almost 99% better than `lFoot, rFoot` and other shortcuts, for example.